stable/artifactory-ha/templates/artifactory-binarystore.yaml

```diff
@@ -24,6 +24,11 @@ data:
             </provider>
         </chain>
         
+        <provider id="cache-fs" type="cache-fs">
```

Comment:
I think this should be wrapped with an if statement. However, if you feel that this is always needed, there should be a default value in values.yaml. In the current state, this can cause issues when you don't pass these values.